### PR TITLE
259 : Add plan_identifier in event metadata table

### PR DESCRIPTION
**Pull request by rehammuzzamil**
Issue #259

### CODE REVIEW

File: assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- // add columns in event metadata table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE core.event_metadata ADD COLUMN plan_identifier VARCHAR DEFAULT NULL;
```
Comment (githengi) [inline assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql:4]: Add an index on it

File: assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+CREATE INDEX event_metadata_plan_identifier_index ON core.event_metadata(plan_identifier);
```
Comment (githengi) [inline assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql:6]: Add a query to populate existing events with the plan Identifier
see related issue https://github.com/OpenSRP/opensrp-server-configs/pull/80#discussion_r449421612

Comment (rehammuzzamil) [reply assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql:6]: Done

File: assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql (hunk 1 above)
Comment (rehammuzzamil) [reply assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql:4]: Done

File: assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+UPDATE core.event_metadata
+SET plan_identifier = (select json ->> 'plan_identifier' from core.event where event.id = event_metadata.event_id);
```
Comment (githengi) [inline assets/migrations/scripts/20200629122125_add_columns_in_event_metadata_table.sql:9]:
```suggestion
SET plan_identifier = (select json->'details'->>'planIdentifier' from core.event where event.id = event_metadata.event_id);
```

**Pull request merged**